tvb_framework/tvb/interfaces/web/controllers/base_controller.py

```diff
@@ -94,6 +94,15 @@ def __init__(self):
                                                   subsection=WebStructure.SUB_SECTION_ALLEN,
                                                   description="Download data from Allen dataset and create a mouse connectome"))
 
+        siibra_algo = self.algorithm_service.get_algorithm_by_module_and_class(
+            IntrospectionRegistry.SIIBRA_CREATOR_MODULE,
+            IntrospectionRegistry.SIIBRA_CREATOR_CLASS
+        )
+        siibra_link = self.get_url_adapter(siibra_algo.fk_category, siibra_algo.id)
+        self.connectivity_submenu.append(dict(title='Siibra Connectivity', link=siibra_link,
```

Comment:
Here we will probably need an IF to check if the siibra-creator can be executed in this installation before displaying the GUI entry.